docs:  local-development (#25394)
renovatebot/renovate: docs/development/local-development.md

docker build -f .devcontainer/Dockerfile -t renovatebot_local .
```

> More recent docker versions are using buildx by default, so you need to run the following to get the image loaded to the docker image store.

Reviewer: ```suggestion
Starting from Docker Engine 23.0 and Docker Desktop 4.19, Docker uses Buildx by default.
So you must run the following command to get the image loaded to the docker image store:

```

Changes:

- Add newline to fix lint error
- Capitalize proper nouns like `Docker` and `Buildx`
- Say which versions use Buildx by default [^1]
- Split sentence into two parts

[^1]: https://docs.docker.com/build/architecture/ lists which versions use Buildx by default.

docker build -f .devcontainer/Dockerfile -t renovatebot_local .
```

Starting from Docker Engine 23.0 and Docker Desktop 4.19, Docker uses Buildx by default.

> So you must run the following command to get the image loaded to the docker image store:

Reviewer: ```suggestion
So you must run the following command to get the image loaded to the docker image store:

```

> So you must run the following command to get the image loaded to the docker image store:

Reviewer: ```suggestion
So you must run the following command to get the image loaded to the Docker image store:

```

Let's try that again. 😄

> ```

Reviewer: I don't know why GitHub strips my newlines. You'll have to put the newline in yourself. 😉
